### ISSUE TITLE
Image cannot be shown if alt text is empty

### ISSUE THREAD

**Issue by li-boxuan** [tomduck/pandoc-fignos#62]
When using pandoc-fignos, image cannot be shown if alt text is missing.

This would not be rendered:

```
![](image.png)
```

Alt text is a must:
```
![alt text is needed](images/knn.png)
```

**Comment by li-boxuan** [comment]
I just removed the generated pdf file and rebuild, the figure appears. Probably something wrong with my `pandoc`, or there is some caching mechanism I am not aware of. Not an issue with `pandoc-fignos`. Sorry for that.
